resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
 
-![](/resources_pages/imgs/Rtools.png)
+![](/resources_pages/imgs/add-conda-env-to-ps1.png)
 
-To see if you were successful, open RStudio and type ```install.packages("xtable", type="source")``` into the console panel. If this install command runs successfully, Rtools should be installed!
+If you type
 
+```
+python --version
+```
 
-### IR kernel
+you should now see the same output as above:
+
+```
+Python 3.8.3
+```
 
-Open RStudio and type the following commands into the console panel:
+And if you type
 
 ```
-install.packages(c('repr', 'IRdisplay', 'evaluate', 'crayon', 'pbdZMQ', 'devtools', 'uuid', 'digest', 'tidyverse'))
-devtools::install_github('IRkernel/IRkernel')
-IRkernel::installspec()
+conda --version
 ```
 
-To see if you were successful, try running Jupyter Lab and seeing if you have working R kernel. To launch the Jupyter Lab type the following in the Windows Powershell:
+you should see something like this
 
 ```
-jupyter lab
+conda 4.8.3
 ```
 
-A browser should have launched and you should see a page that looks like the screenshot below. Now click on "R" notebook (circled in red on the screenshot below) to launch an Jupyter Lab with an R kernel.
+> Note that if you want to run Python directly from the command line on Windows, you need to append the `-i` flag, so the full command would be `python -i`. Running just `python` will freeze the Git Bash terminal.
 
-![](/resources_pages/imgs/jupyter_lab_r_kernel.png)
+One annoyance with our current terminal setup is that the word `(base)` is not on the same row as the rest of the prompt string (the part with `your_name@your_computer`. To fix this we can edit the `.bash_profile` configuration file to indicate that we do not want a newline at the beginning of the prompt string. Open up the configuration file using VS Code by typing the following command into a terminal:
 
-Sometimes a kernel loads, but doesn't work as expected. To test whether your installation was done correctly now type `library(tidyverse)` in the code cell and click on the run button to run the cell. If your R kernel works you should see something like the image below:
+```
+code "/c/Program Files/Git/etc/profile.d/git-prompt.sh"
+```
 
-![](/resources_pages/imgs/jupyter_lab_r_kernel2.png)
+Delete the line that reads the following (it should be line 13):
 
-### Rscript
+```
+PS1="$PS1"'\n'       # new line
+```
 
-To be able to call R scripts from the Powershell or Git Bash you need to be able to execute the Rscript program from there. To make this possible you need to add the the file path of the directory/folder that houses the Rscript program (i.e., location where this program lives on your computer) to the Windows PATH system variable. You can modify the PATH system variable using System Utility in control panel on Windows. We outline below how to do this:
+Click to save the file,
+when VS Code prompts you that the saving failed,
+click "Retry as Admin" and then "Yes".
```

Comment:
This is a really nice function of vscode, never have to right click and "open as admin"!